cmd/debug.go

```diff
@@ -20,8 +20,13 @@ func newDebugCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// debug Cmd represents the debug command
 	var debugCmd = &cobra.Command{
 		Use:   "debug",
-		Short: "Run the local Appsody environment in debug mode",
-		Long:  `This starts a docker based continuous build environment for your project with debugging enabled.`,
+		Short: "Debug your Appsody project",
+		Long:  `Start a container-based continuous build environment for your Appsody project, with debugging enabled.`,
+		Example: `  appsody debug --docker-options "--privileged"
+  Starts debugging environment and passes the "--privileged" option to the docker run as a flag.
```

Comment:
Need more articles, ie "the debugging environment". What do you mean by "docker run"? Do you mean "the Docker "run" command", or "the "docker run" command", or something else?

Reply:
I kept leaving out articles to make it fit better on the CLI - I will push these changes but might need help to make these example explanations shorter!

Reply:
Ah ok, didn't realise there was a length issue! You could maybe have "Starts the debugging environment, passing the "--privileged" option to the docker run as a flag.", though that's only slightly shorter.

Reply:
Thanks! I have pushed the chnages